client/src/main/scala/com/all4journey/webapp/pages/PersonalInfoFormJsImpl.scala

```diff
@@ -50,7 +48,7 @@ object PersonalInfoFormJsImpl extends PersonalInfoFormJs  with AddressTypePickle
   }
 
   @JSExport
-  def personalInfoForm(user: User, homeAddress: Option[Address], token:String) = div(cls := "container")(
+  def personalInfoForm(user: User, token:String) = div(cls := "container")(
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Method is longer than 50 lines](https://www.codacy.com/app/rjkernick/all4journey/file/2737533537/issues/source?bid=3246460&fileBranchId=3352852#l51)